tooling/v8-snapshot/src/blueprint/custom-require.js

```diff
@@ -226,7 +226,7 @@ function resolvePathsAndParent (
   }
 
   if (snapshotting || !modulePathFromAppRoot.startsWith('.')) {
-    filename = modulePathFromAppRoot
+    filename = modulePathFromAppRoot.split('/').join(PATH_SEP)
```

Comment:
```suggestion
    // Have to hard code / here since we can't use `path` in the snapshot
    filename = modulePathFromAppRoot.split('/').join(PATH_SEP)
```